src/components/containers/NewToDoContainer.ts

```diff
@@ -2,8 +2,13 @@ import { connect } from "react-redux";
 import { addToDo } from "../../redux/toDoSlice/toDoSlice";
 import NewToDo from "../NewToDo/NewToDo";
 import { ToDo } from "../../types/types";
+import { Dispatch } from "redux";
 
-const mapDispatchToProps = (dispatch: any) => {
+export type NewToDoContainerProps = {
```

Comment:
why did you created the prop type in container file.

If you are only using them in component file make the type in that composnnt not in container